resources/definitions/fdmprinter.def.json

```diff
@@ -1166,6 +1166,17 @@
                     "limit_to_extruder": "wall_0_extruder_nr",
                     "settable_per_mesh": true
                 },
+                "z_seam_relative":
+                {
+                    "label": "Z Seam Relative",
+                    "description": "When enabled, the z seam coordinates are relative to each part's centre. By default, the coordinates define an absolute position on the platform.",
```

Comment:
The description should be independent from the default value, because it might be different for each printer.
Suggested: "By default" => "When disabled"

also "platform" => "build plate"